### ISSUE TITLE
Økofysiologi

### ISSUE THREAD

**Issue by kyrretl** [realfagstermer/realfagstermer#1571]
Føler kanskje det er litt merkelig at dette emneordet er overorndet Tilpasning/Adaptasjon siden man kan ha tilpasninger som ikke er rent økofysiologiske (som typer morfologiske og adferdsmessige tilpasninger f.eks.)

---
*REAL001609 (Økofysiologi) i [Emnesøk](https://app.uio.no/ub/emnesok/realfagstermer/search?id=c001609), [Skosmos](http://data.ub.uio.no/realfagstermer/c001609), [Okapi](http://ub-viz01.uio.no/okapi2/#/search?q=realfagstermer%3A%22%C3%98kofysiologi%22), [Soksed](http://ub-soksed.uio.no/concepts/REAL001609)*

**Comment by BioHeidi** [comment]
Sammenslåinger basert på belegg. Men om du vil splitte og rydde er det bare fint!

**Comment by Hypsibius** [comment]
Kan vi ta opp igjen denne? Har eit dokument her om asymmetri og 'handedness', mao anatomiske tilpassingar (om ein antar adaptiv signifikans). 

https://bibsys-almaprimo.hosted.exlibrisgroup.com/permalink/f/1cruloh/BIBSYS_ILS71468818670002201

**Comment by Hypsibius** [comment]
![image](https://user-images.githubusercontent.com/9350370/71616960-189b5a80-2bb9-11ea-9189-38894ae970bf.png)

![image](https://user-images.githubusercontent.com/9350370/71617008-539d8e00-2bb9-11ea-9910-e7dd9a1aca57.png)

![image](https://user-images.githubusercontent.com/9350370/71617032-74fe7a00-2bb9-11ea-93ec-596662ea670d.png)

![image](https://user-images.githubusercontent.com/9350370/71617105-c4dd4100-2bb9-11ea-9c2d-c21492809527.png)

**Comment by Caroline-A** [comment]
Jeg lurer på om det har vært en misforståelse med sammenslåing her, fordi siste gang vi snakket om den ble forslag å rydde opp i BF men beholde to termer: Økofysiologi (BF Fysiologiske tilpassningar osv), og så Tilpasning (BF Miljøtilpasninger (flyttet fra økofysiologi) osv.) - re issue #1446

**Comment by danmichaelo** [comment]
Hm, @BioHeidi slo dem sammen 12. desember 2018, men svarte ikke på #1446 da, og markerte den heller ikke som lukket, så tipper hun ikke så GitHub-saken før hun slo sammen.

Vi kan godt splitte hvis du vil være med å rydde @Hypsibius ! @kyrretl , ser de foreslåtte definisjonene greie ut for deg?

**Comment by kyrretl** [comment]
Ser greit ut, har egentlig bare en kommentar på navngiving

Vi kan ha "Tilpasninger" fremfor "Evolusjonære tilpasninger" som foretrukken term? Altså at det er underforstått at "Tilpasninger" er reservert for evolusjonære prosesser?

 Jmf Hypsibius' post

Har også en tanke om at definisjon på Tilpasninger også bær belyse at organismer kan ha tilpasninger som ikke nødvendigvis gir økt fitness i dagens miljø, men som ga økt fitness statistisk over tid i dets fortidsmiljø (EEA - Environment of Evolutionary Adaptedness).

**Comment by Hypsibius** [comment]
Litt uviss på Kyrres siste ønske, det _kan_ skape forvirring, og den som søker på tilpassing vil få treff på 'Evolusjonær tilpassing':  Både Akkomodasjon og adaptasjon forklarast m tilpassing. Mrk også forma 'Tilpassing' også på Bokm

![image](https://user-images.githubusercontent.com/9350370/75431853-c3c45800-594d-11ea-9dd1-2492bb2dee46.png)

![image](https://user-images.githubusercontent.com/9350370/75431947-ea828e80-594d-11ea-9648-811f46c6338d.png)

**Comment by kyrretl** [comment]
Ja, men hverken humord eller realfagstermer har akkommodasjon som term eller BF-term.
Hvis vi ser på ordet Tilpasning i wikipedia f.eks. så er den eneste artikkelen om dette tilpasning fra et biologisk perspektiv.

Men dette belyser vel et generelt spørsmål om hvor generelle vi kan være i et fagvokabular fremfor regulære ordbøker. Jeg har egentlig ingen sterke preferanser her, ser Hypsibius poeng også, ville bare høre med dere hvordan vi ser på fagområder som "eier" visse ord.

**Comment by Hypsibius** [comment]
Veit! Og kjenner det klør litt etter å plante evolusjonsflagget på denne. Men her er jo både akkomodasjon (typ augets tilpassing til mørke) og adaptasjon begge biologiske termar då. Her er det evolusjonslære mot fysiologi, liksom ;-) Når eg blir irettesatt med den språkbruken, er det sjeldan av ikkje-biologar, forråsresånn

**Comment by danmichaelo** [comment]
Som ikke-biolog har jeg ikke noe imot at "Tilpasninger" blir oppslagsordet, så lenge vi legger på definisjon :) Kan du oppsummere hvor vi står nå, @kyrretl ?

**Comment by kyrretl** [comment]
Ehm, kan jeg det?

Kan komme med et forslag og så kan vi se om de andre biologene er enige:

Vi lager Tilpasninger BF Adaptasjoner BF Evolusjonære tilpasninger BF Evolusjonære adaptasjoner .

Her er kortfattet forsøk på definisjon, gjerne endre der det trengs!

Tilpasninger:
"Tilpasninger innenfor biologien er en arts egenskaper og adferd som gjør den vellykket i det miljøet den har levd i over evolusjonær tid. Tilpasningene er gradvis formet av evolusjonen gjennom naturlig utvalg."

Kommentar for videre arbeid(?)
Når det gjelder Akkomodasjoner er det flott emneord som skiller fysiologi fra evolusjon som Hypsibius nevnte. Vi har ikke Akkomodasjoner som term og jeg mistenker at Tilpasninger eller Adaptasjoner kan ha blitt satt feil, sånn som Utviklingsbiologi kontra Evolusjonsbiologi kan ha blitt feilsatt. Men her skal jeg ikke åpne en diger ryddeprosses altså!
